.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ jobs:
 
 # sanity
   test-sanity-check:
-    resource_class: medium+
+    resource_class: xlarge
```

Comment:
Hmm, is that needed?

Reply:
Yep, I checked medium+ and large resource classes and they did not have enough memory for sanity check.